tox.ini

```diff
@@ -24,29 +24,36 @@ extend-exclude = docs/notebooks/,.venv
 
 [testenv]
 basepython = python3.7
-deps = pip~=20.3
+deps = pip
```

Comment:
will this potentially mean an old pip version will be used?

Reply:
This should use the latest version (which we were explicitly doing anyway in most of the commands via a pip install).